exporter/signalfxexporter/translation/converter.go

```diff
@@ -113,13 +114,11 @@ func (c *MetricsConverter) metricDataToSfxDataPoints(md consumerdata.MetricsData
 	//  Q.: what about resource type?
 	nodeAttribs := md.Node.GetAttributes()
 	resourceAttribs := md.Resource.GetLabels()
-	numExtraDimensions := len(nodeAttribs) + len(resourceAttribs)
-	var extraDimensions []*sfxpb.Dimension
-	if numExtraDimensions > 0 {
-		extraDimensions = make([]*sfxpb.Dimension, 0, numExtraDimensions)
-		extraDimensions = appendAttributesToDimensions(extraDimensions, nodeAttribs)
-		extraDimensions = appendAttributesToDimensions(extraDimensions, resourceAttribs)
-	}
+
+	extraDimensions := make([]*sfxpb.Dimension, 0, len(nodeAttribs)+len(resourceAttribs)+1)
+
+	extraDimensions = appendResourceAttributesToDimensions(extraDimensions, resourceAttribs)
+	extraDimensions = appendAttributesToDimensions(extraDimensions, nodeAttribs)
```

Comment:
nit: would be nice to have `appendNodeAttributesToDimensions` for consistency

Reply:
have given it a second thought, we can keep it as is since the function generic itself. so up to you